WcaOnRails/app/views/delegate_reports/_form.html.erb

```diff
@@ -0,0 +1,11 @@
+<%= simple_form_for(@delegate_report) do |f| %>
+  <%= f.error_notification %>
+
+  <div class="form-inputs">
+    <%= f.input :content %>
+  </div>
+
+  <div class="form-actions">
+    <%= f.button :submit %>
+  </div>
+<% end %>
```

Comment:
Ah, good catch. There is no `new` action for delegate reports, only edit. Then, when you're done with your report, you post it. This is leftover from when @viroulep and I started work on this in April, I honestly totally forgot this existed.